liquidation-monitor/src/agent.js

```diff
@@ -0,0 +1,500 @@
+          );
+        }
+        /* eslint-enable no-param-reassign */
+        return null;
+      }));
+    }));
+    // #endregion
+
+    // #region Update all token prices via 1inch for now.
+    ts('Updating token prices and collateral factors');
+    // To-Do: Switch the primary feed back to ChainLink
+    await Promise.all(Object.keys(data.tokens).map(async (currentToken) => {
+      const price = await oneInchContract.getRateToEth(data.tokens[currentToken].underlying, 0);
+      // Adjust for native decimals
+      const oneInchMult = BigNumber(10).pow(36 - data.tokens[currentToken].tokenDecimals);
+      /* eslint-disable no-param-reassign */
+      data.tokens[currentToken].price = BigNumber(price.toString())
+        .dividedBy(oneInchMult);
+      // Update the Collateral Factor
+      const market = await comptrollerContract.markets(currentToken);
+      data.tokens[currentToken].collateralMult = BigNumber(market[1].toString())
+        .dividedBy(BigNumber(10).pow(18));
+      /* eslint-enable no-param-reassign */
+      ts(
+        `Updated price of ${data.tokens[currentToken].symbol
+        } is ${data.tokens[currentToken].price.toString()
+        } ETH with a collateral factor of ${data.tokens[currentToken].collateralMult}`,
+      );
+    }));
+    // #endregion
+
+    // #region Calculate health on all accounts
+    ts('Health recalculation');
+    Object.keys(data.accounts).forEach((account) => {
+      let borrowBalance = BigNumber(0);
+      let supplyBalance = BigNumber(0);
+      Object.keys(data.tokens).forEach((token) => {
+        if (data.supply[token][account]) {
+          // Supply balances are stored in cTokens and need to be multiplied by the
+          //   exchange rate and reduced by the collateral factor
+          supplyBalance = supplyBalance.plus(data.supply[token][account]
+            .multipliedBy(data.tokens[token].exchangeRateMult)
+            .multipliedBy(data.tokens[token].price)
+            .multipliedBy(data.tokens[token].collateralMult));
+        }
+        if (data.borrow[token][account]) {
+          // Only need to multiply by the price.
+          borrowBalance = borrowBalance.plus(data.borrow[token][account]
+            .multipliedBy(data.tokens[token].price));
+        }
+      });
+
+      // Remove non-borrowers
+      if (borrowBalance.eq(0)) {
+        /* eslint-disable no-param-reassign */
+        delete data.accounts[account];
+      } else {
+        data.accounts[account].supplyBalance = supplyBalance;
+        data.accounts[account].borrowBalance = borrowBalance;
+        data.accounts[account].health = supplyBalance.dividedBy(borrowBalance);
+        /* eslint-enable no-param-reassign */
+      }
+    });
+    // #endregion
+
+    // #region Check for low health accounts currently health <1.03
+    const lowHealthAccounts = [];
+    Object.keys(data.accounts).forEach((currentAccount) => {
+      if (data.accounts[currentAccount].health.isLessThan(1.03)) {
```

Comment:
Make the health threshold a parameter that is specified in the `agent-config.json`.